core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -692,7 +692,12 @@ private[spark] class DAGScheduler(
     }
 
     val jobId = nextJobId.getAndIncrement()
+    val startTime = clock.getTimeMillis()
```

Comment:
Nit: put this inside the if statement. It's not just the start time; maybe call it "time" as it's used as the end time too.

Reply:
Nice